### ISSUE TITLE
Tota11y Check - Profile page

### ISSUE THREAD

**Issue by arandomdude24** [elenirotsides/Team-418#91]
Profile page: /profile

Headings
- Profile page title needs to be [h1] tag

Contrast
- All blue buttons have insufficient contrast ratio
